File: Panel_Gestion/src/main/java/com/sanivoice/gestion_interna/Facultativo.java
package com.sanivoice.gestion_interna;

public class Facultativo {
	private String nombre,
				   apellidos,
				   numColegiado,
				   especialidad,
				   nombreUsuario,
				   clave;

	public Facultativo(String nombre, String apellidos, String numColegiado, String especialidad, String nombreUsuario,
			String clave) {
		super();
		this.nombre = nombre;
		this.apellidos = apellidos;
		this.numColegiado = numColegiado;
		this.especialidad = especialidad;
		this.nombreUsuario = nombreUsuario;
		this.clave = clave;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getApellidos() {
		return apellidos;
	}

	public void setApellidos(String apellidos) {
		this.apellidos = apellidos;
	}

	public String getNumColegiado() {
		return numColegiado;
	}

	public void setNumColegiado(String numColegiado) {
		this.numColegiado = numColegiado;
	}

	public String getEspecialidad() {
		return especialidad;
	}

	public void setEspecialidad(String especialidad) {
		this.especialidad = especialidad;
	}

	public String getNombreUsuario() {
		return nombreUsuario;
	}

	public void setNombreUsuario(String nombreUsuario) {
		this.nombreUsuario = nombreUsuario;
	}

	public String getClave() {
		return clave;
	}

	public void setClave(String clave) {
		this.clave = clave;
	}
	
	
}

File: ServicioSaludNoAdscrito/Facultativo.php
<?php
class Facultativo implements JsonSerializable{
    private $id;
    private $nombre;
    private $apellidos;
    private $especialidad;

    public function __construct($id, $nombre, $apellidos, $especialidad){
        $this->id=$id;
        $this->nombre=$nombre;
        $this->apellidos=$apellidos;
        $this->especialidad=$especialidad;
    }

    public function jsonSerialize() {
        return (object) get_object_vars($this);
    }

    public function getId(){
        return $this->id;
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function getApellidos(){
        return $this->apellidos;
    }

    public function getEspecialidad(){
        return $this->especialidad;
    }
}



?>